packages/cozy-harvest-lib/src/services/budget-insight.spec.js

```diff
@@ -60,7 +60,8 @@ const account = {
   _id: '1337',
   auth: {
     login: '80546578',
-    secret: 'secretsecret'
+    secret: 'secretsecret',
+    bankId: TEST_BANK_COZY_ID
```

Comment:
This does not seem to be tested, should we check that it is correctly passed to createTemporaryToken ?

Reply:
CreateTemporaryToken has an assert for the presence of this attribute. And had to make this modification to allow the test to pass